starsessions/session.py

```diff
@@ -72,7 +72,10 @@ def get_session_remaining_seconds(connection: HTTPConnection) -> int:
     """Get total seconds remaining before this session expires."""
     now = time.time()
     metadata = get_session_metadata(connection)
-    return int((metadata["created"] + metadata["lifetime"]) - now)
+    # use "last_update" if rolling session, otherwise use "created"
+    rolling = metadata.get("rolling", False)
+    last_update = metadata["last_update"] if rolling and "last_update" in metadata else metadata["created"]
```

Comment:
What if we will use `last_access` everywhere? This seems very natural to me. Rolling session is alive while it's being used. So `last_access` may work.

Reply:
I believe we should use `last_access` here, as the purpose of rolling session is not to expire while being used.
`last_access` is the timestamp when the session was last loaded, read used.

I think that we should not use `last_updated` to count session remaining time because I think it is a good idea not to call `handler.save` at all if session was not modified to avoid network round trip, e.g. with Redis backend (feature out of scope).

Since we always update `last_access` on every load (read endpoint call), we do extend rolling session TTL by this action. So, my opinion is just to use `last_access` in place of `created` and this should solve your requirement.